front-end/src/App.css

```diff
@@ -2,27 +2,12 @@
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.recordBtn {
```

Comment:
As much as we all love camelcase, I think dashes would be more idiomatic for our CSS (pardon the music pun). Not a big deal as long as we're consistent.